app/views/accounts/edit.html.slim

```diff
@@ -16,28 +16,29 @@
                   i.fa.fa-camera
                 span.file-label
                   | Add photo
-      .field
+      .field.mb-4
         = f.text_field :name, autofocus: true, class: "input", placeholder: "Name"
-        br
-        br
       .field
-        = f.check_box :notification, class: "checkbox", id: 'notification-check-box'
-        |  &nbsp;
+        = f.check_box :notification, class: "checkbox mr-2", id: 'notification-check-box'
         = f.label :notification, "Report transactions"
-      .field
-        = f.email_field :email, disabled: !account.notification, autocomplete: "email", id:"email-input",  placeholder: "email"
+      .field.mb-4
+        = f.email_field :email, disabled: !account.notification, autocomplete: "email", id:"email-input",  placeholder: "email", required: true
         .notification.is-light.mt-1
-          | You can check the box and specify the email of the person for whom you have created this account to send them notifications about balance changes
-      br
+          | You can check the box and specify your kid's email to send them notifications about balance changes.
+      .field.mb-4
+        = f.check_box :notice_to_parents, class: "checkbox mr-2"
+        = f.label :notice_to_parents, "Report transactions to parents"
+        .notification.is-light.mt-3
+          | You and all users to whom you've shared access to your kid's account can receive email notifications about balance changes. By default, this option is enabled, but you can disable it.
```

Comment:
If you have given access to your child's account to other users, including yourself, you have the option to receive email notifications about balance changes. By default, this feature is turned on, but it can be disabled if you prefer not to receive these notifications.

Reply:
@VladislavSokov hold on please with the message. I'm working on it.

Reply:
How about this:

> If enabled, all users with access to your child's account, including you, will get email notifications about balance changes.